### DISCUSSION TITLE
quay.io 25 june 2022 update influence

### DISCUSSION THREAD

**Discussion by usefree** [prometheus-operator/prometheus-operator#4824] [General]
Hello everybody. There is the announce on quay.io site
"On June 15, 2022, Repository Signing and App Registry will be removed from quay.io. Users should migrate to use alternatives which are now natively supported (Cosign & Helm respectively)"
Please, clarify, if it affect in some way the source of images, hosted on its container registry?
for example 
quay.io/prometheus/node-exporter
quay.io/prometheus-operator/prometheus-config-reloader
quay.io/prometheus/alertmanager
Seems it has no effect, but just to be sure
Thanks!

**Comment by simonpasquier** [comment]
:wave: nothing that should have an impact on the prometheus-operator or prometheus repositories :)
